disco_aws_automation/disco_deploy.py

```diff
@@ -50,10 +51,13 @@ def _get_hostclasses_from_pipeline_definition(self, pipeline_definition):
         ''' Return hostclasses from pipeline definitions, validating numeric input '''
         hostclasses = {entry["hostclass"]: entry for entry in pipeline_definition}
 
-        for hostclass, entry in hostclasses.iteritems():
-            for definition in ["max_size", "desired_size", "min_size"]:
-                if definition in entry:
-                    hostclasses[hostclass][definition] = int(entry[definition])
+        for entry in hostclasses.itervalues():
+            if "min_size" in entry:
+                entry["min_size"] = int(DiscoAWS._size_as_minimum_int_or_none(entry["min_size"]))
```

Comment:
This seems like a generic enough function that it can be moved to disco_aws_utils. '_' prefix in a function name is used to indicate a private function in Python world.